src/app/shared/components/create-table-modal/create-table-modal.component.ts

```diff
@@ -38,6 +42,9 @@ import { widthTypes } from '@shared/messages/col-width-types';
 export class CreateTableModalComponent implements OnInit {
   private modalService = inject(ModalService);
   private formBuilder = inject(FormBuilder);
+  private fs = inject(Firestore);
+  private authService = inject(AuthService);
+  authUser!: User | null;
```

Comment:
```suggestion
  authUser: User | null = null;
```
if it's User or null why not to define it as null?